### ISSUE TITLE
Globs within comments

### ISSUE THREAD

**Issue by Nirazul** [tomgrooffer/gulp-sass-glob#16]
Globs within line comments get parsed like that:

```scss
// @import "util/functions/**/*.scss";
```

```scss
// @import "util/functions/_list-contains.scss";
@import "util/functions/_str-split.scss";
@import "util/functions/_strip-unit.scss";
```

This is probably causing the bug: `/@import\s+["']([^"']+\*(\.scss|\.sass)?)["'];?/`

I would love to fix the used regex to exclude those matches, but I haven't found any useful regex patterns and I'm not good enough at finding a solution myself :/

Maybe this will help you as a starting point: https://regex101.com/r/kH8mZ0/2

**Comment by CREEATION** [comment]
Didn't look in the plugin code itself, but this regex works for me on regex101.com:

```js
/^\s*@import\s+["']([^"']+\*(\.scss|\.sass)?)["'];?$/gm
```

https://regex101.com/r/vU9oX5/1

**Comment by Nirazul** [comment]
That should do the trick!
@tomgrooffer What do you think?

**Comment by tomgrooffer** [comment]
@CREEATION thanks man, @Nirazul on it!

**Comment by tomgrooffer** [comment]
@CREEATION much love and thanks :) fixed, update to v1.06
